flake.nix

```diff
@@ -32,89 +81,173 @@
         import nixpkgs {
           inherit system;
           overlays = [ self.overlay ];
+          config.android_sdk.accept_license = true;
         });
 
     in {
 
       # A Nixpkgs overlay.
-      overlay = final: prev: {
-
-        hello = with final;
-          stdenv.mkDerivation rec {
-            name = "hello-${version}";
-
-            src = hello-src;
-
-            buildInputs =
-              [ autoconf automake gettext gnulib perl gperf texinfo help2man ];
-
-            preConfigure = ''
-              mkdir -p .git # force BUILD_FROM_GIT
-              ./bootstrap --gnulib-srcdir=${gnulib-src} --no-git --skip-po
-            '';
+      overlay = final: prev:
+        with final.pkgs; {
+
+          sdk = pkgs.androidenv.composeAndroidPackages {
+            buildToolsVersions = [ "28.0.3" "29.0.3" ];
+            platformVersions = [ "29" ];
+            abiVersions = [ "x86" "x86_64" "armeabi-v7a" "armeabi-v8a" ];
+            includeNDK = true;
+            ndkVersion = "22.0.7026061";
+          };
+          ndk = sdk.ndk-bundle;
+          buildPythonPackage = python3.pkgs.buildPythonPackage;
+          zxcvbn = python3.pkgs.zxcvbn;
+          androidSystem = androidSystemByNixSystem.${system};
```

Comment:
is this a necessary output at all?

Reply:
No, you're right. I'll move it to where it is needed.